collector/main.go

```diff
@@ -3,15 +3,22 @@ package main
 import (
 	"github.com/Kindling-project/kindling/collector/application"
 	"log"
+    "github.com/Kindling-project/kindling/collector/version"
 )
 
 func main() {
 	app, err := application.New()
 	if err != nil {
 		log.Fatalf("Failed to create application: %v", err)
 	}
+	
+	//print version information
+	log.Printf("GitCommitInfo:%s\n", version.Version())
```

Comment:
I would rather move this to the beginning of the main function in which way even if users can not create `application`, we could also know the version.